### ISSUE TITLE
Pal sending me signal messages, I don't have it, it says delivered

### ISSUE THREAD

**Issue by lansalot** [WhisperSystems/Signal-Android#4373]
So, I had an Experia Z1 months back. I tried TextSecure over a year ago and thought it was OK, but I didn't really commit and so removed it.

Factory-erased it a couple months back and sold it, bought a Galaxy S6. Didn't install Signal at all. When I look in the play store, it says it's installed. Doesn't say where tho. I'm allowed to install it and it offers my Galaxy. The Experia no longer shows in my devices as of course I removed it. To be clear - Signal is absolutely NOT installed on my Galaxy and never has been.

He texted me yesterday via signal and it said it was delivered securely. I didn't receive it (of course, I don't have Signal). So I texted him a normal SMS today and it came up on his device with the little padlock. How is this? I can't possibly have sent him a secure message, and where are his messages to me going?

When he goes to message me, it says "Send Signal message", which would have led him to believe it was encrypted and being sent to me.

I've kept my same mobile number, and the buyer of my experia didn't get any old sim cards or anything. So, ummm... bug?

**Comment by piratenpanda** [comment]
Try unregistering here:
https://whispersystems.org/textsecure/unregister/

**Comment by lansalot** [comment]
Tried that. Their page appears to be having "issues". When I requested the code and did "Send SMS", the page came back with a tag saying "Failed to connect to the server". Note, that's not a browser message, but text on the page, under "unregister".

Also, while it might fix our problem, the fact is surely the system should be smart enough to know it hasn't been used for well over a year? It's telling him the message was delivered securely too. So surely that's wrong.
